docs/markdown/using-snowchain/validating-and-relaying/relayer-architecture.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Relayer Architecture
+parent: Validating and Relaying
+permalink: /relayer-architecture/
+nav_order: 3
+---
+
+# Relayer Architecture
+The relayer is a single golang binary that acts as a worker container for various workers that are involved in the relaying process. As our bridge requires a mix of different kinds of data types and proofs to be relayed at different times, co-ordinated in different ways, we split up these responsibilities into different workers.
+
+## Worker Goals
+ - Each worker is expected to be stateless and functional
```

Comment:
This seems to be wrong – workers are not functional in the "no side effect" sense, they interact through networking and a database and maintain internal state. 

Maybe I'm misunderstanding your point?

Reply:
yeh, fixing..